internal/locale/locales/en-us.yaml

```diff
@@ -334,9 +334,7 @@ err_api_forbidden:
 err_api_org_not_found:
   other: Unable to find requested Organization
 err_api_project_not_found:
-  other: The requested project [NOTICE]{{.V0}}[/RESET] does not exist under [NOTICE]{{.V1}}[/RESET]. Please ensure the owner is correct and that the project has been created.
-err_api_project_not_found_unauthenticated:
-  other: The requested project [NOTICE]{{.V0}}/{{.V1}}[/RESET] could not be found
+  other: The requested project [NOTICE]{{.V1}}[/RESET] does not exist under [NOTICE]{{.V0}}[/RESET]. Please ensure the owner is correct and that the project has been created.
```

Comment:
nit: Why are the variables here out of order?

Reply:
To keep the order consistent in the code itself, see lines 43 and 46 here:

https://github.com/ActiveState/cli/blob/d657e19c76b2d44150c02d7327361ba21591ed00/pkg/platform/model/projects.go#L43-L46